Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs

```diff
@@ -169,6 +169,30 @@ public async Task Test_Update_File_UpdateMany()
         Assert.IsTrue(GetDocuments("update").Equals(true));
```

Comment:
https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L5
This namespace is not used, so it can be removed.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L54
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L55
Please use `IsFalse` instead of `IsTrue`.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L72
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L73
No need to check if equal. You can just assert the return of the method.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L90
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L91
No need to check if equal. You can just assert the return of the method.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L111
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L112
No need to check if equal. You can just assert return of the method.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L132
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L133
No need to check if equal. You can just assert return of the method.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L150
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L151
No need to check if equal. You can just assert return of the method.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L168
Please split this into their own asserts.

https://github.com/FrendsPlatform/Frends.MongoDB/blob/d0e8ea9bd527ce05cae71d48044848c473b4e3c3/Frends.MongoDB.Update/Frends.MongoDB.Update.Tests/UnitTests.cs#L169
No need to check if equal. You can just assert return of the method.